app/adapters/application.js

```diff
@@ -1,5 +1,5 @@
 import DS from 'ember-data';
 
 export default DS.RESTAdapter.extend({
-    host: 'https://api.webmaker.org'
+    host: 'http://localhost:2015'
```

Comment:
Out of sheer curiosity, does ember support some way of injecting config into the build process?